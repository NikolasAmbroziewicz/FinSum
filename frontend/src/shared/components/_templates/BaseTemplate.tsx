export interface IBaseTemplate {
  sampleTextProp: string;
}

const BaseTemplate: React.FC<IBaseTemplate> = ({ sampleTextProp }) => {
  return (
    <div className="bg-gradient-to-r from-cyan-500 to-blue-500 underline text-black flex">
      {sampleTextProp}
    </div>
  );
};

export default BaseTemplate;
