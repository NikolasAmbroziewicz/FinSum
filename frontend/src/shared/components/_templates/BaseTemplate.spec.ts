import { describe, it, expect } from 'vitest';
import BaseTemplate, { IBaseTemplate } from './BaseTemplate';

const templateProps: IBaseTemplate = {
  sampleTextProp: 'text'
};

describe('BaseTemplate', () => {
  it('defined', () => {
    expect(BaseTemplate(templateProps)).toMatchSnapshot();
  });
});
