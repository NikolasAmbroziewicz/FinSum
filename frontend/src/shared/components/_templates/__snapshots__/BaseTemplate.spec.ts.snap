// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BaseTemplate > defined 1`] = `
<div
  className="bg-gradient-to-r from-cyan-500 to-blue-500 underline text-black flex"
>
  text
</div>
`;
